ci_framework/roles/manage_secrets/README.md

```diff
@@ -0,0 +1,44 @@
+# manage_secrets
+Copy secret content such as pull-secret and ci-token to a (not really) secure
+location with appropriate rights.
+
+## Privilege escalation
+None
+
+## Parameters
+* `cifmw_manage_secrets_basedir`: (String) Base directory for the directory tree. Default to `cifmw_basedir` which defaults to `~/ci-framework-data`.
+* `cifmw_manage_secrets_pullsecret_dest`: (String) Destination file for pull-secret.json. Defaults to `{{ cifmw_manage_secrets_basedir }}/secrets/pull_secret.json`.
+* `cifmw_manage_secrets_pullsecret_file`: (String) Absolute path to the source pull-secret. Defaults to `null`.
+* `cifmw_manage_secrets_pullsecret_content`: (String) Content of the source pull-secret. Defaults to `null`.
+* `cifmw_manage_secrets_citoken_dest`: (String) Destination file for ci-token. Defaults to `{{ cifmw_manage_secrets_basedir }}/secrets/ci_token`.
+* `cifmw_manage_secrets_citoken_file`: (String) Absolute path to the source ci-token. Defaults to `null`.
+* `cifmw_manage_secrets_citoken_content`: (String) Content of the source ci-token. Defaults to `null`.
+* `cifmw_manage_secrets_kube_dest`: (String) Destination directory for kube configuration files. Defaults to `~/.kube`.
+* `cifmw_manage_secrets_kube_type`: (String) Type of kube configuration file. Must be either `config` or `kubeadmin-password`. Defaults to `null`.
+* `cifmw_manage_secrets_kube_file`: (String) Absolute path to the source kube configuration file. Defaults to `null`.
+* `cifmw_manage_secrets_kube_content:` (String) Content of the kube configuration file. Defaults to `null`.
+
+## File versus Content
+You can set either a file XOR a content for the pull-secret and ci-token secrets. You can't set both.
```

Comment:
XOR :) but really, docs should be accessible/easy to read so imo jusr 'OR' is better here

Reply:
XOR is nice :). But probably not known enough - even ansible doesn't support it ;_;